portal-frontend/src/app/features/applications/view-submission/view-application-submission.component.html

```diff
@@ -126,5 +114,47 @@ <h2>Applicant Submission</h2>
         </div>
       </mat-tab>
     </mat-tab-group>
+    <div *ngIf="application && application.canEdit" class="continue-button-container">
+      <button mat-stroked-button color="warn" (click)="onCancel(application.uuid)">
+        Cancel Application
+      </button>
+      <button mat-flat-button color="primary" *ngIf="application && application.canEdit" 
+        [routerLink]="'/application/' + application.fileNumber + '/edit'">
+        <div class="continue-button-content">
+          <span>Continue Application</span>
+          <mat-icon>keyboard_double_arrow_right</mat-icon>
+        </div>
+      </button>
+    </div>
+    <div *ngIf="application && application.canReview" class="continue-button-container">
+      <button mat-stroked-button color="warn" (click)="onCancel(application.uuid)">
+        Cancel Application
+      </button>
+      <button (click)="onReview(application.fileNumber)"  mat-flat-button color="primary"
+        *ngIf="
+          application.status.code === SUBMISSION_STATUS.IN_REVIEW_BY_LG || 
+          application.status.code === SUBMISSION_STATUS.RETURNED_TO_LG || 
+          application.status.code === SUBMISSION_STATUS.SUBMITTED_TO_LG
+        ">
+        <div class="continue-button-content">
```

Comment:
I think `div` inside of `button` violates HTML standard. Is this div really needed?

Reply:
we need that div, since we want it as button with arrows